src/datasets/meshes/mpi_faust.jl

```diff
@@ -0,0 +1,161 @@
+
+The dataset required to be downloaded manually from the [website](http://faust.is.tue.mpg.de/) and extracted in the correct location. For information about where to place the dataset, refer to the example section.
+
+# Dataset Variables
+
+- `scans`: Vector of non-watertight scans in the form of `Mesh`.
+- `registrations`: Vector of registrations corresponding to each scan in `scans`. `registrations` like `scans` are also in the form of `Mesh`.
+- `labels`: For each scan in the training set, we provide the boolean Vector of length equal to the number of vertices in the corresponding scan. It represents which vertices were reliably registered by the corresponding registration.
+- `metadata`: A dictionary containing additional information on the dataset. Currently only `:test` split has metadata containing information about the registrations required for the inter and intra challenge proposed by the author.
+
+# Examples
+
+## Loading the dataset
+
+```julia-repl
+julia> using MLDatasets
+
+julia> dataset = FAUST()
+[ Info: This program requested access to the data dependency MPI-FAUST
+[ Info: It could not be found on your system. It requires manual installation.
+┌ Info: Please install it to one of the directories in the DataDeps load path: /home/user/.julia/packages/DataDeps/EDWdQ/deps/data/MPI-FAUST,
+│ /home/user/.julia/datadeps/MPI-FAUST,
+│ /home/user/.julia/juliaup/julia-1.7.3+0.x86/local/share/julia/datadeps/MPI-FAUST,
+│ /home/user/.julia/juliaup/julia-1.7.3+0.x86/share/julia/datadeps/MPI-FAUST,
+│ /home/user/datadeps/MPI-FAUST,
+│ /scratch/datadeps/MPI-FAUST,
+│ /staging/datadeps/MPI-FAUST,
+│ /usr/share/datadeps/MPI-FAUST,
+└ or /usr/local/share/datadeps/MPI-FAUST
+[ Info: by following the instructions:
+┌ Info: Dataset: MPI-FAUST.
+└ Website: http://faust.is.tue.mpg.de/
+Once installed please enter 'y' reattempt loading, or 'a' to abort
+[y/a]
+```
+Now download and extract the dataset into one of the given locations. For unix link systems, an example command can be
+```bash
+unzip -q <path-to-filename</filename.zip ~/.julia/datadeps
+```
+The corresponding folder tree should look like
+```
+├── test
+│   ├── challenge_pairs
+│   └── scans
+└── training
+    ├── ground_truth_vertices
+    ├── registrations
+    └── scans
+```
+Press `y` to re-attept loading.
+```julia-repl
+dataset FAUST:
+  scans          =>    100-element Vector{Any}
+  registrations  =>    100-element Vector{Any}
+  labels         =>    100-element Vector{Vector{Bool}}
+  metadata       =>    Dict{String, Any} with 0 entries
+```
+
+## Load train and test split
+
+```julia-repl
+julia> train_faust = FAUST(:train)
+julia> test_faust = FAUST(:test)
```

Comment:
```suggestion
julia> train_faust = FAUST(:train);

julia> test_faust = FAUST(:test);
```
julia-repl blocks should match what you would see in the repl